k8s/k8s.go

```diff
@@ -161,6 +167,37 @@ type DaemonSetOps interface {
 	UpdateDaemonSet(*apps_api.DaemonSet) error
 }
 
+// JobOps is an interface to perform job operations
+type JobOps interface {
+	// CreateJob creates the given job
+	CreateJob(job *batch_v1.Job) (*batch_v1.Job, error)
+	// GetJob returns the job from given namespace and name
+	GetJob(namespace, name string) (*batch_v1.Job, error)
```

Comment:
I minor thing - everywhere else we are passing name as the first param followed by namespace. Would be good to keep it consistent in all the newly added methods.